src/shared/test/integration/rpc-bridge-test.js

```diff
@@ -0,0 +1,185 @@
+import Bridge from '../../bridge';
+
+describe('RPC-Bridge integration', () => {
+  const sandbox = sinon.createSandbox();
```

Comment:
I'm not sure if we actually need to use a custom sandbox here. This would normally only be needed if you wanted to temporarily patch properties of global objects (eg. a `window` method) for the duration of a test.

Reply:
We don't use the sandbox at all 😄 .

Because this integration test derives from a unit test, I forgot to remove it.